README.md

```diff
@@ -1 +1,139 @@
-# math-magician1
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
- [optional] great job. you can follow this [README](https://github.com/microverseinc/readme-template) design so you can add the table of contents. nice work